### ISSUE TITLE
Getting 'no scheduler for type' in deploying a container from the REST API

### ISSUE THREAD

**Issue by rydhar** [shipyard/shipyard#571]
Hi 

I am new to shipyard and I am trying to run the a deploy command that has worked via the CLI now through the REST API. Below is an example with the service key and IP address mocked. When I run the command I get the following:

HTTP/1.1 500 Internal Server Error
Content-Type: text/plain; charset=utf-8
Date: Thu, 09 Jul 2015 18:14:24 GMT
Content-Length: 23
Proxy-Connection: keep-alive
Keep-Alive: 60
Via: HTTP/1.1 proxy1732

no scheduler for type 

CLI command
shipyard run --name dockerregistry.tools.nikecloud.com/gatling –-vol --label test1 /opt/gatling/user-files:/opt/gatling/user-files –-arg /opt/gatling/user-files/heartbeat.sh

REST API call
curl -D- 'http://10.0.0.109:8080/api/containers' -X POST -H 'X-Service-Key: 1234567890' -H 'Content-Type: application/json' --data '{ "name": "dockerregistry.tools.nikecloud.com/gatling","labels": ["test1"],"args": ["/opt/gatling/user-files/heartbeat.sh"], "links": { "/opt/gatling/user-files":"/opt/gatling/user-files" }}'

Am I using one of the keys improperly in the JSON body?

**Comment by ehazlett** [comment]
This should work.  It may be a bug in the request for the cpu / mem
setting.  I'll take a look.

**Comment by rydhar** [comment]
I think I was able to specify "type":"service" in the JSON body. I realized that I was misusing the links I think because I am getting the following error:

curl -D- 'http://10.0.0.146:8080/api/containers' -X POST -H 'X-Service-Key: 1234567890' -H 'Content-Type: application/json' --data '{ "name": "dockerregistry.tools.nikecloud.com/gatling", "type": "service", "labels": ["test1"], "args": ["/opt/gatling/user-files/heartbeat.sh"],"links": { "/opt/gatling/user-files": "/opt/gatling/user-files" } }'
HTTP/1.1 500 Internal Server Error
Content-Type: text/plain; charset=utf-8
Date: Mon, 13 Jul 2015 23:56:40 GMT
Content-Length: 80
Proxy-Connection: keep-alive
Keep-Alive: 60
Via: HTTP/1.1 proxy1732

500 Internal Server Error: Could not get container for /opt/gatling/user-files

Per the CLI docs : "--link: link another container"

I actually want to use : "--vol: container volumes (/host/path:/container/path or /container/path)"

There is no example in your REST API docs for this. Is it simply replacing "links" with "vol"?
